index.js

```diff
@@ -37,11 +37,7 @@ var query = function(path, done) {
         });
 
         res.on('end', function() {
-            try {
-                done(false, output, path);
-            } catch(e) {
-                done(e, output, path);
-            }
+            done(false, output, path);
```

Comment:
You're right... The try..catch block does not make sense in this context :-1: 
Thanks for your review :+1: